cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -395,6 +396,23 @@ func (m *AwsManager) buildNodeFromTemplate(asg *asg, template *asgTemplate) (*ap
 	node.Status.Capacity[gpu.ResourceNvidiaGPU] = *resource.NewQuantity(template.InstanceType.GPU, resource.DecimalSI)
 	node.Status.Capacity[apiv1.ResourceMemory] = *resource.NewQuantity(template.InstanceType.MemoryMb*1024*1024, resource.DecimalSI)
 
+	if asg.MixedInstancesPolicy != nil && asg.MixedInstancesPolicy.instanceRequirementsOverrides != nil {
+		instanceReqirements := asg.MixedInstancesPolicy.instanceRequirementsOverrides
+
+		if instanceReqirements.VCpuCount != nil {
+			if instanceReqirements.VCpuCount.Min != nil {
+				node.Status.Capacity[apiv1.ResourceCPU] = *resource.NewQuantity(*instanceReqirements.VCpuCount.Min, resource.DecimalSI)
+			}
+		}
+
+		if instanceReqirements.MemoryMiB != nil {
+			if instanceReqirements.MemoryMiB.Min != nil {
+				node.Status.Capacity[apiv1.ResourceMemory] = *resource.NewQuantity(*instanceReqirements.MemoryMiB.Min*1024*1024, resource.DecimalSI)
+			}
+		}
```

Comment:
Observing the other values used in node-building on lines 393-396, I chose to keep only similar values from the instance requirements, inferring this as the other requirements specs not being relevant for Cluster Autoscaler's scale-up calculations. 
All of these values will be read from the instance type selected from GetInstanceTypesFromInstanceRequirements API in `aws_wrapper.go`, so overriding only provides value in ensuring minimum specs are used in calculations.